devserver.js

```diff
@@ -1,4 +1,4 @@
-var spawn = require('child_process').spawn;
+var spawn = require('cross-spawn').spawn;
```

Comment:
no need for cross-spawn, i think the minimum version of node to run fauxton is 4